src/app/components/calendar/calendar.ts

```diff
@@ -1465,12 +1470,12 @@ export class Calendar implements OnInit, OnDestroy, ControlValueAccessor {
         }
     }
 
-    decrementDecade() {
-        this.currentYear = this.currentYear - 10;
+    decrementYearpickerstep() {
+        this.currentYear = this.currentYear - this.stepYearpicker;
     }
 
-    incrementDecade() {
-        this.currentYear = this.currentYear + 10;
+    incrementYearpickerstep() {
```

Comment:
same here, `incrementYearPickerStep`

Reply:
I changed it to "incrementYear**P**icker**S**tep". ("decrement-" and "stepYearpicker" analoge)

I am not sure about the wording. Maybe "incrementYearPicker" (without "-step") would be better? Or maybe "incrementYearPickerRange" or "incrementYearPickerValues"?